instructure/canvas-ios
# Tutorial for QR based Parent account creation and Parent sign in

refs: MBL-14291
affects: Parent
release note: none

test plan, 

1. check all the qr code flags
2. click on qr login > I don't have a canvas account
3. make sure the tutorial is shown
4. click on qr login > I have a canvas account
5. make sure the old tutorial is shown
